### ISSUE TITLE
the orange favicon is missing in the documentation

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-flutter#371]
As a user of the documentation
I want ta view the documentation
So that it is identified as an orange documentation in the browser identification icon

The orange favicon should added in the top index page folder: /favicon.ico

[favicon.ico.zip](https://github.com/user-attachments/files/22406877/favicon.ico.zip)

**Comment by B3nz01d** [comment]
closed during sprint # 22
